README.md

```diff
@@ -29,13 +29,30 @@ Until now, it is being developed and tested using Google Chrome browser.
     $ rosdep install web_video_server
     ```
 
-3. Launch rosbridge websocket server and web video server
+
+4. Install latest development version of rosbrige_server [you can skip this, but then the ActionLib Widget is not working]
```

Comment:
I dont get it...
first install normal version, then install git dev version (so all dependencies are there)